### ISSUE TITLE
Subfolders for playlists

### ISSUE THREAD

**Issue by knochenhans** [emoon/HippoPlayer#142]
I really love in DeliPlayer that you can put tracks into subfolders. This is also used when you scan folders with subdirectories which are then reflected in the resulting playlist, makes dealing with huge playlists much more convenient.

**Comment by emoon** [comment]
How do you mean put tracks into subfolders? Is these virtual subfolders (i.e doesn't exist on disk) more like a structure within a playlist?

**Comment by knochenhans** [comment]
Exactly! Virtual tree style folders, like seen in this shot (even featuring subfolders for recognized sub tracks)

![image](https://user-images.githubusercontent.com/5293125/89510389-cc5c6c80-d7d0-11ea-96a9-728dea750798.png)

**Comment by emoon** [comment]
Cool. Yeah I have thought about having sub songs as "sub tree" to the main song. This is something that will behave similar when zip support and similar things is added is my plan.
